subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/tasks/ReleaseNeeded.java

```diff
@@ -28,11 +33,24 @@ boolean releaseNeeded(ReleaseNeededTask task, EnvVariables envVariables) {
         boolean releaseNeeded = releaseNeed.needed;
         String message = releaseNeed.explanation;
 
-        if (!releaseNeeded && task.isExplosive()) {
-            throw new GradleException(message);
-        } else {
-            LOG.lifecycle(message);
+        File releaseNeededFile = new File(task.getProject().getBuildDir(), RELEASE_NEEDED_FILENAME);
+
+        if (releaseNeededFile.exists()) {
```

Comment:
Great! We need to delete the file.

Can you make it the first thing that happens in this task?